.github/CODEOWNERS

```diff
@@ -3,11 +3,11 @@
 /src/lib/autocomplete/**                           @crisbeto
 /src/lib/badge/**                                  @amcdnl
 /src/lib/bottom-sheet/**                           @jelbourn @crisbeto
-/src/lib/button-toggle/**                          @tinayuangao
-/src/lib/button/**                                 @tinayuangao
+/src/lib/button-toggle/**                          @jelbourn
```

Comment:
@jelbourn Now that I have really looked into checkbox and button, I can take on ownership of them if you would like

Reply:
Done